themes/bootstrap3/templates/ajax/status-available-services.phtml

```diff
@@ -0,0 +1,23 @@
+<?
+// defaultServiceStatusMessage for multiple status and as fallback for missing translations
+$defaultServiceStatusMessage = 'HoldingStatus::services_available_html';
+
+// generate serviceStatusMessage
+$serviceStatusMessage =
+    count($services) == 1
+        ? $this->transEsc('HoldingStatus::service_available_'.$services[0],[],$defaultServiceStatusMessage)
+        : $defaultServiceStatusMessage ;
+
+// build the translated serviceList for usage in defaultServiceStatusMessage
+$that = $this;
+$translator = function ($in) use ($that) {
+    return '<span class="service_' . $in . '">' . $that->transEsc('HoldingStatus::service_' . $in) . '</span>';
```

Comment:
By changing `$that->transEsc('HoldingStatus::service_' . $in)` into `$that->transEsc('HoldingStatus::service_' . $in, [], $in)` we should be good with unanticipated services without translatable string.